Profile Card Using ReactJS

Kalyani02072003 opened the pull request:
## Description 📜

Added a ReactJs project to make a profile card.
Fixes #63 

<hr />

## Type of change 📝

<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->

- [x] New feature (non-breaking change which adds functionality)

<hr />

## Domain of Contribution 📊

<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->

- [x] HTML, CSS, JS
- [x] React


<hr />
 
## Checklist ✅

<!----Please delete options that are not relevant. And in order to tick the check box just but x inside them for example [x] like this----->

- [x] I follow [Code of conduct](https://github.com/surajm-333/Ace-The-FrontEnd/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added required screenshot/gif depicting the working of the project in the folder
- [x] I am a GSSoC'22 contributor

<hr />

<!----Please delete options that are not relevant. And in order to tick the check box just but x inside them for example [x] like this----->

## Screenshots / Gif 📸
![Screenshot (308)](https://user-images.githubusercontent.com/71889838/156377235-709833a8-4f64-43d7-a68c-74460de064c6.png)

![Screenshot (309)](https://user-images.githubusercontent.com/71889838/156377255-01d73269-30b1-4b13-bab2-ad395aeb7fb6.png)

surajm-333 commented:
@Kalyani02072003 add the screenshots in the project folder. Update the readme file of this particular project and include project description and screenshot in it. You can refer [this](https://github.com/surajm-333/Ace-The-FrontEnd/blob/main/CryptoRush-Application/Cryptoworld/README.md)

surajm-333 commented:
@Kalyani02072003 change the pr title and keep an appropriate one. Also you should add demo1.png and demo2.png as screenshots in project folder. It is already mentioned in previous comment.

Kalyani02072003 commented:
> @Kalyani02072003 change the pr title and keep an appropriate one. Also you should add demo1.png and demo2.png as screenshots in project folder. It is already mentioned in previous comment.

sorry sir I misunderstood the screenshot part, added the screenshots now and changed the pr title

Kalyani02072003 commented:
@surajm-333 hello sir, is there anything else i need to do or is it fine? please have a look at it

Kalyani02072003 commented:
> @Kalyani02072003 getting this error when running the project ![image](https://user-images.githubusercontent.com/66875689/156620668-0acbaf38-9e5f-4a3f-81b9-253caa6cf2aa.png)

it should be fontawesome instead of fortawesome, how can i edit the files?

surajm-333 commented:
@Kalyani02072003 you need to create a new pull request suggesting appropriate changes in this project folder.

Kalyani02072003 commented:
> @Kalyani02072003 you need to create a new pull request suggesting appropriate changes in this project folder.

Okay i will create a new pull request. Changes will be suggested by you right ?

surajm-333 commented:
@Kalyani02072003 no, you need to edit the files in current folder only. You need to apply changes that you have mentioned and make sure to test it once so that everything is working fine, before making a pull request.
